app/views/play/cinematic/cinematicController.js

```diff
@@ -41,13 +42,13 @@ export class CinematicController {
     this.stage.addChild(this.backgroundAdapter.container)
     this.stage.addChild(this.layerAdapter.container)
 
-    // TODO: Will be moved to camera commands.
-    this.systems.camera.zoomTo({ x: 0, y: 0 }, 6, 0)
+    camera.zoomTo({ x: 0, y: 0 }, 6, 0)
```

Comment:
New to the cinematic system design (have read both of the docs you posted in Slack), so this might not make sense.  If the CameraSystem is responsible for setting up / managing the camera, would it make sense to have this enqueued as the first command by the the camera system?  If this works it would allow us to keep all of the camera functionality in one place.

Reply:
Also - can we test that the initial camera position / zoom is initialized properly?

Reply:
Great call out. Refactoring to move this into the camera system creation. I don't think it necessarily fits in the commands. But it definitely doesn't fit here!
Thanks!